### ISSUE TITLE
Agenda for Jan 26, 2023

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/interop#265]
Here's the proposed agenda for our meeting on Thursday, Jan 26:

- Dashboard review
- Identifying remaining tasks and owners:
  - Labeling tests
  - Proposal feedback wording
- Confirming date for launch

Previous meeting: https://github.com/web-platform-tests/interop/issues/262

**Comment by chrishtr** [comment]
Attendees: @foolip @bkardell @chrishtr @tantek @jensimmons @jgraham @nt1m @boazsender @zcorpan @robnyman @meyerweb @dandclark 
(apologies if I missed anyone)

Notes:

* Launch date
  * Discussions seem to be going well, almost there finding an agreed date
* Dashboard
  * Looking at dashboard prototype together
  * Brian: It seems unintuitive how you can see browser-specific numbers in the table that are higher than the Interop score next to them - a drive-by reader would be confused
  * “Interop” means passing in all browsers, which is why the score is lower
  * Boaz: It also took me a while to understand how the numbers in the circles were computed
  * Brian: Can we put in a venn diagram to explain?
  * Boaz: Make the circles in the second line smaller?
* Jen: I have some feedback on the dashboard, but also we need to decide on a launch date etc
  * Philip: Please file issues on repo for desired improvements at https://github.com/web-platform-tests/wpt.fyi/issues with [Interop 2023] in description
  * Philip: Let’s talk about launch date
  * Chris: Feb 2 seems to be the consensus candidate for date
  * Robert: Let’s summarize blocking issues that must be done by Monday. Let’s fix blocking issues but can also fix lower priority issues after launch if necessary.
  * Philip: Confident that we can fix most bugs in the dashboard in time to unblock launch.
* Consensus: Everyone agrees that Feb 2 is the launch date
* Philip: Time of day for announcement?
  * Robert: propose 9am PST.
  * Resolved to 9am PST.
  Robert: Now let’s enumerate blockers.
  * Jen: Blog posts written, responses to excluded proposals finished, dashboard polished
  * Philip: Finish labeling tests also. Think we still need work for web compat and web codecs
  * James: Still a PR or two to add tests for web compat, but otherwise I consider that good enough
  *Consensus: Labeling is good enough to launch, not blocking
* Brian: I like the idea of a joint post explaining Interop 2023. Boaz has a draft document. Can we have such a thing and link to it from the dashboard?
* Philip: Propose updating the readme to include this information.
* Boaz: I can add prose to the readme. [Philip’s PR](https://github.com/web-platform-tests/interop/pull/266)
* Philip will land a PR to briefly update the readme to what interop 2023 is, and then Boaz to write another PR to explain the focus areas etc
* Chris: Deadline to finish text for all responses to excluded proposals done by EOD Monday in the timezone of the person doing the work. Then Philip will ask anyone who has not checked the checkboxes for the responses needs to add a comment proposing changes or check the box, by EOD Tuesday.
* Chris: How will we share blog posts drafts to each other? Like last year?
* Robert: We have a document to share draft blog posts. We’ll work from the model of “it’s ok unless anyone says something”.
* Chris: Propose that EOD Friday is the deadline to file GitHub issues for any blocking issues for the launch.
* Jen: Agreed. Will file some issues related to accessibility in particular.
* Boaz: Have some thoughts about ways to improve the information architecture, e.g. consistently labeling stuff on top.
* Jen: Maybe photoshop up some ideas?
* Boaz: I’ll do that.
* Tim: FYI I also landed a CL to make the colors for engines on the various graphs be consistent across the wpt.fyi site
  * Philip, others: Probably want to change some of the colors for better visibility and accessibility
  Maybe change the Chrome color to some kind of yellow - goldenrod?
  * Tim to write a PR to change the colors of all graphs to be Chrome yellow, Firefox red/orange, Safari blue, interop ??
